Fix Help Button in User create page

skks1212 opened the pull request.

nihal467 commented:
@skks1212 
![image](https://user-images.githubusercontent.com/57055998/179754263-6883a9b2-aaa7-4333-afa5-3b204c7fca9a.png)

having this much space covered with need help doesn't give a good look, can we have an alternative design

nihal467 commented:
@skks1212 
![image](https://user-images.githubusercontent.com/57055998/185943685-612e4725-52bb-434a-9115-e5bb73d64ccf.png)

- [ ] change the button to the right side of the same line where the add user is located
- [ ] change the colour from violet to grey

skks1212 commented:
@nihal467 I have changed the button position, but gray color was looking a bit dull and seemed like the button was disabled.
![image](https://user-images.githubusercontent.com/23238460/186144646-b09f92c7-892d-4348-bc25-63accb7b139b.png)

Thus i changed the color to green.
![image](https://user-images.githubusercontent.com/23238460/186145300-b893aa98-0f2c-4f41-b2c4-016b55f00aac.png)

nihal467 commented:
@skks1212 can you ask @gigincg or post in slack

skks1212 commented:
![image](https://user-images.githubusercontent.com/23238460/186212467-dad98c48-7096-4088-89de-2b1c9c565145.png)
@nihal467 I had a talk with Gigin, he has suggested to keep it gray

nihal467 commented:
test approved
